coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -69,11 +69,12 @@ RDEPEND=">=sys-apps/baselayout-3.0.0"
 # If protocol 1 is installed it must be configured to provide the default
 # implementation based on whether protocol 2 is enabled or not.
 RDEPEND="${RDEPEND}
-	etcd_protocols_1? (
-		dev-db/etcd:1
-		!etcd_protocols_2? ( dev-db/etcdctl )
-	)
-	etcd_protocols_2? ( dev-db/etcd:2 )
+	etcd_protocols_1? ( dev-db/etcd:1 )
+	etcd_protocols_2? (
+        dev-db/etcd:2
+        app-admin/etcd-wrapper:2
```

Comment:
I promise we use extensive use of profiles. But it is up to ebuilds to have correct dependencies for any given arch. You can use a hacky workaround by adding the etcd wrapper to "package.provided" in the arm profile which makes it basically go away but that's not what package.provided is really for and should be used sparingly.